plugins/flutter_aepcore/android/src/main/java/com/adobe/marketing/mobile/flutter/flutter_aepcore/FlutterAEPCoreDataBridge.java

```diff
@@ -118,6 +124,36 @@ static String stringFromPrivacyStatus(final MobilePrivacyStatus privacyStatus) {
         return AEP_PRIVACY_STATUS_UNKNOWN;
     }
 
+    static InitOptions initOptionsFromMap(final Object option) {
```

Comment:
nit: Can you fix the variable names.
option -> arguments
optionsAsMap -> argumentsAsMap
initOptionMap -> initOptionsMap